src/components/new-request/search.vue

```diff
@@ -440,75 +478,19 @@ export default class NewSearch extends Mixins(CommonMixin) {
   }
 
   get isFederal () {
-    return (this.location === Location.CA && this.jurisdiction === Location.FD)
+    return (this.getLocation === Location.CA && this.getJurisdictionCd === Location.FD)
   }
 
   get isInternational () {
-    return (this.location === Location.IN)
+    return (this.getLocation === Location.IN)
   }
 
-  // If selected radio button is Named Company.
+  /** Whether selected radio button is Named Company. */
   get isNamedCompany (): boolean {
     return (this.selectedCompanyType === CompanyType.NAMED_COMPANY)
   }
 
-  get isPersonsName () {
```

Comment:
Some of these (below) weren't used at all. Others I replaced with direct store getter/action calls.